sdk/cosmos/azure-cosmos-spark_3-5_2-12/README.md

```diff
@@ -0,0 +1,249 @@
+# Azure Cosmos DB OLTP Spark 3 connector
+
+## Azure Cosmos DB OLTP Spark 3 connector for Spark 3.5
+**Azure Cosmos DB OLTP Spark connector** provides Apache Spark support for Azure Cosmos DB using
+the [SQL API][sql_api_query].
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows
+developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+If you have any feedback or ideas on how to improve your experience please let us know here:
+https://github.com/Azure/azure-sdk-for-java/issues/new
+
+### Documentation
+
+- [Getting started](https://aka.ms/azure-cosmos-spark-3-quickstart)
+- [Catalog API](https://aka.ms/azure-cosmos-spark-3-catalog-api)
+- [Configuration Parameter Reference](https://aka.ms/azure-cosmos-spark-3-config)
+
+[//]: # (//TODO: add more sections)
+[//]: # (//TODO: Enable Client Logging)
+[//]: # (//TODO: Examples)
+[//]: # (//TODO: Next steps)
+[//]: # (//TODO: Key concepts)
+[//]: # (//TODO: Azure Cosmos DB Partition)
+[//]: # (//TODO: Troubleshooting)
+
+### Version Compatibility
+
+#### azure-cosmos-spark_3-5_2-12
```

Comment:
Readme.md needs to be updated for other Spark versions following the current pattern as well. But this can for sure happen in separate PR - and I think we should see whether we can avoid the need to manually update the readmes with every release. So, adding the comment only as reminder for all of us - no need to react to it in this PR - just before next release

Reply:
Just made a pr for this.